src/V3GraphAlg.cpp

```diff
@@ -112,15 +108,15 @@ class GraphAlgRemoveTransitiveEdges final : GraphAlg<> {
         : GraphAlg<>(graphp, nullptr) {}
     void go() {
         GraphPathChecker checker{m_graphp};
-        for (V3GraphVertex* vxp = m_graphp->verticesBeginp(); vxp; vxp = vxp->verticesNextp()) {
+        for (V3GraphVertex& vtx : m_graphp->vertices()) {
             V3GraphEdge* deletep = nullptr;
-            for (V3GraphEdge* edgep = vxp->outBeginp(); edgep; edgep = edgep->outNextp()) {
-                if (deletep) VL_DO_CLEAR(deletep->unlinkDelete(), deletep = nullptr);
+            for (V3GraphEdge& edge : vtx.outEdges()) {
+                if (deletep) VL_DO_DANGLING(deletep->unlinkDelete(), deletep);
```

Comment:
DO_CLEAR means always do the zeroing, DO_DANGLING it's "optional" and disabled for some formal analysis.  This code requires deletedp to be cleared as is checked later.

Reply:
Thanks! Fixed.